ui/react-ui/src/components/routes/reports/userIdReport/UserIdTasksReport.js

```diff
@@ -0,0 +1,28 @@
+
+import React, { useState, useEffect } from 'react';
+import { JsonToTable } from "react-json-to-table";
+
+function TasksReport() {
+    const [jsonReport, setJsonReport] = useState('');
+    const jsonUrl = "https://luca-artillery-report.s3-us-west-2.amazonaws.com/report__for_30req.json";
+    const proxy  = 'https://cors-anywhere.herokuapp.com/';
```

Comment:
not sure if you need this line, it should probably work without this line.

Reply:
Strangely I couldnt get the request without doing this , Im still understanding what CORS is but thats really the reason why its there.

Reply:
I bumped into that issue too. I was wondering how come @ahmadafrasiyab didn't have this proxy in his code and it was working properly. also, I noticed RGs in Azure have this feature to configure cors to accept requests from localhost:3000, not sure about AWS though.

Reply:
Right?? I was wondering the same thing.

Reply:
I was just reviewing another, and was using localhost:3001 and also got cors issues. I switched to 3000 and it worked.

Reply:
Oh  very interesting! I was only using 3000 for this feature.

Reply:
It could be related to the port. I think react runs on 3000 as a standard?